src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -98,7 +101,7 @@ export default class SaveInProgressIntro extends React.Component {
     const savedForm = profile && profile.savedForms
       .filter(f => moment.unix(f.metadata.expires_at).isAfter())
       .find(f => f.form === this.props.formId);
-    const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
+    const prefillAvailable = this.props.prefillAvailable || !!(profile && profile.prefillsAvailable.includes(this.props.formId)); // TODO: remove 1st clause once 526 added to list
```

Comment:
If the plan isn't to add it to the list now, can you write a ticket to cover that work when we're ready? (Including changing what needs to be changed here.)